specs/person_spec.rb

```diff
@@ -0,0 +1,21 @@
+require_relative '../person'
+
+describe Person do
+  describe '#can_use_services?' do
+    it 'returns true if the person is of age' do
+      person = Person.new(25)
+
+      result = person.can_use_services?
+
+      expect(result).to be true
+    end
+
+    it 'returns true if the person has parent permission' do
+      person = Person.new(16, 'John', parent_permission: true)
+
+      result = person.can_use_services?
+
+      expect(result).to be true
+    end
```

Comment:
- [ ] @nilab-khwaja and @Izebiafe Kindly consider adding tests for all methods in the Person class, you are missing tests for `correct_name`, `add_rental` and `to_json`, kindly consider adding them:

![image](https://github.com/nilab-khwaja/OOP-school-library/assets/61408860/78df33f8-ce97-4681-9005-dff0e65fd636)